### PR TITLE
Humans can no longer be martians

### PR DESCRIPTION
**Pull request by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE**
This is obviously a bug and caused the last runtimes flood back in December of last year that made pomf's computer grow sentience. Don't expect any controversial opinions about removing this

Uses process() to generate a semifold matrix that gets rid of martian humanoidification. Still can be commented out if shitmins feel like it.
Probe came into my house last night and beat me up so I also don't feel like changelogging this one

### CODE REVIEW

**Comment by DamianX** [comment]
but what does it look like?

**Comment by Irradiation** [comment]
HE DID IT AGAIN

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
In like 80% of the rooms in the station where machinery gives off a bit of lighting it looks pretty good
dreamseeker_2018-09-02_23-33-57

But in absolute total darkness it causes a tiny square of softlighting because of g**nlights and there's no way around it that I know, except maybe when we move to 512
dreamseeker_2018-09-02_23-33-05

**Comment by kilozombie** [comment]
HE DID IT
THE MADMAN

File: code/game/machinery/lightswitch.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	anchored = 1.0
 	var/buildstage = 2
 	var/on = 0
+	var/image/overlay
```
Comment (DamianX) [inline code/game/machinery/lightswitch.dm:12]: you could make this `static`
edit: oh, you'd have to make two, one for the on state and one for the off state
or you could just not do it

**Comment by SonixApache** [comment]
meme description, no changelog

File: code/game/machinery/lightswitch.dm [hunk 2]
```diff
@@ -29,10 +30,22 @@
 	updateicon()
 
 /obj/machinery/light_switch/proc/updateicon()
-	if ((stat & NOPOWER) || buildstage != 2)
+	if(!overlay)
+		overlay = image(icon, "light1-overlay")
+		overlay.plane = LIGHTING_PLANE
+		overlay.layer = ABOVE_LIGHTING_LAYER
```
Comment (DamianX) [inline code/game/machinery/lightswitch.dm:37]: image

Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [reply code/game/machinery/lightswitch.dm]: It's identical to how borg eyes work

You can only see overlays if you can see the base object

**Comment by Kurfursten** [comment]
I will only give this PR my :+1: if you give light switches the name var they've deserved for years now. It's time to stop being "machinery".

**Comment by SonixApache** [comment]
reeee

**Pull request merged**